pysat/tests/test_instrument.py

```diff
@@ -559,15 +546,16 @@ def test_incorrect_creation_empty_instrument_object(self):
 
     @raises(AttributeError)
     def test_supplying_instrument_module_requires_name_and_platform(self):
-        class Dummy: pass
+        class Dummy:
+            pass
         Dummy.name = 'help'
 
         temp = pysat.Instrument(inst_module=Dummy)
 
 
 class TestBasicsXarray(TestBasics):
     def setup(self):
-        reload(pysat.instruments.pysat_testing)
+        re_load(pysat.instruments.pysat_testing)
```

Comment:
All of these re_loads aren't really correct due to my previous generalization that wasn't quite complete. To be consistent, it should reallybe a reload of pysat_testing_xarray, but then that module doesn't get tested properly as noted in comment above.

Reply:
Enhancement noted by issue #182